src/harness/testcases/testdata/fdb_7/FDB_7_fcc_id.csv

```diff
@@ -0,0 +1 @@
+fccId,maxEirp

```

Comment:
This is not known at this time, please remove.

Reply:
removed. How shall we deal with this then ?

Reply:
The current plan is to leave FDB.7 out of the May 28 pack. We are waiting for the FCC to determine the format and I was advised earlier that test dev should not wait for the FCC on this matter. I have an email with FCC and other SAS Admins (Masoud and/or Kurt should be CCed) about this issue and asking what we should do after May 28.